src/utils/git_hub_path.cr

```diff
@@ -2,6 +2,6 @@ module GitHubPath
   extend self
 
   def for_file(file_path = __FILE__)
-    "https://github.com/paulcsmith/website-v2/blob/master#{file_path.gsub(Dir.current, "")}"
+    "https://github.com/luckyframework/website/blob/master#{file_path.gsub(Dir.current, "")}"
```

Comment:
I think this caused issues because Dir.current changed after compilation or something. But I had an idea for it: `...master#/src/pages#{file_path.split("src/pages", 2).last}`. This way it splits at src/pages and we just ignore where the current directory is during compile